dot/parachain/types/bitvec.go

```diff
@@ -82,6 +71,46 @@ func bytesToBits(b []byte, size uint) []bool {
 			bits = append(bits, bit)
 		}
 	}
+	bv.bits = bits
+}
+
+// MarshalSCALE fulfils the SCALE interface for encoding
+func (bv BitVec) MarshalSCALE() ([]byte, error) {
+	buf := bytes.NewBuffer(nil)
+	encoder := scale.NewEncoder(buf)
+	if len(bv.bits) > 268435455 {
+		return nil, fmt.Errorf("bitvec too long")
```

Comment:
sentinel error?

Reply:
Do you think we'll ever need to `errors.Is` this error? It's only produced once as well, so I left it as inline error.

Reply:
I ended up bringing back the sentinel error. Forgot to the length check when decoding.